lib/mycha_installer.coffee

```diff
@@ -0,0 +1,67 @@
+async = require 'async'
+{exec} = require 'child_process'
+fsExtra = require 'fs-extra'
+path = require 'path'
+
+
+class MychaInstaller
+
+  constructor: ({@cwd, @testHelperPath}) ->
+
+
+  install: (done) ->
+    async.parallel [
+      @_installDependencies
+      @_writeMychaConfig
+      @_writeTestHelper
+    ], done
+
+
+  _installDependencies: (done) =>
+    exec "npm install --save-dev chai sinon sinon-chai", {@cwd}, done
+
+
+  _writeMychaConfig: (done) =>
```

Comment:
Okay, yeah will look at generators in a follow up PR. 
For now are we okay with it just overwriting the file if it exists? 
Should I add some output so the user knows that the files are written?
```shell
$ mycha install
prompt: Default test helper path: spec/spec_helper.coffee

npm install chai sinon sinon-chai # in bold
# npm install output

create spec/spec_helper.coffee
create mycha.coffee
```